cypher/cypher-docs/src/docs/dev/syntax/values.asciidoc

```diff
@@ -69,13 +69,26 @@ Structural types comprise:
    ** Id
```

Comment:
Just a suggestion in order to avoid the repetition of the verb comprise, to change the `Structural types comprise:` to `Structural types are composed of:`.

Reply:
Comprised of is an expression in English that means "to compose or constitute". While its use is common in writing and speech, it has been regarded by some language professionals as incorrect, stemming from the fact that comprise on its own already means "composed of". - https://en.wikipedia.org/wiki/Comprised_of

I will then remove the word `comprised of` for the `composed of`

Reply:
I don't remember seeing `comprised of`, just `comprise` and `comprising`.

Reply:
True, `comprise` was there, but I have replaced it with `composed of`.

Reply:
My two cents here, there is a difference between _comprise_ and _compose_. It has to do with parts and the whole. The whole _comprises_ a number of parts whereas the parts _compose_ the whole. In this case, _comprise_ is the correct choice, as in _Structural types are comprised of_.